### PR TITLE
doc: warn contrastmgr deprecation in fsl 5.0.7+

### PR DESCRIPTION
**Pull request by mgxd**
#1140

### CODE REVIEW

File: nipype/interfaces/fsl/model.py [hunk 1]
```diff
@@ -1075,13 +1084,17 @@ class ContrastMgr(FSLCommand):
     """Use FSL contrast_mgr command to evaluate contrasts
 
     In interface mode this file assumes that all the required inputs are in the
-    same location.
+    same location. This has deprecated for FSL versions 5.0.7+ as the necessary
+    corrections file is no longer generated by FILMGLS.
     """
 
+    if LooseVersion(FSVersion) >= LooseVersion("5.0.7"):
```
Comment (satra) [inline nipype/interfaces/fsl/model.py]: what if FSL is not installed?

File: nipype/interfaces/fsl/model.py [hunk 2]
```diff
@@ -30,6 +31,14 @@
                     BaseInterfaceInputSpec)
 from .base import FSLCommand, FSLCommandInputSpec, Info
 
+FSVersion = "0"
```
Comment (satra) [inline nipype/interfaces/fsl/model.py]: replace this section with the version property in FSL base class.

File: nipype/interfaces/fsl/model.py (hunk 1 above)
Comment (mgxd) [inline nipype/interfaces/fsl/model.py]: then FSVersion would be "0"

File: nipype/interfaces/fsl/model.py [hunk 3]
```diff
@@ -1075,13 +1072,16 @@ class ContrastMgr(FSLCommand):
     """Use FSL contrast_mgr command to evaluate contrasts
 
     In interface mode this file assumes that all the required inputs are in the
-    same location.
+    same location. This has deprecated for FSL versions 5.0.7+ as the necessary
+    corrections file is no longer generated by FILMGLS.
     """
-
+    if Info.version() and LooseVersion(Info.version()) >= LooseVersion("5.0.7"):
+        warn("ContrastMgr is deprecated in FSL 5.0.7+", DeprecationWarning)
```
Comment (satra) [inline nipype/interfaces/fsl/model.py]: you can use DeprecationWarning without import warn

**Pull request merged**